src/LodeRunner.API/src/Handlers/ResultHandler.cs

```diff
@@ -16,17 +17,20 @@ public static class ResultHandler
         /// <summary>
         /// ContentResult factory.
         /// </summary>
-        /// <param name="message">string.</param>
-        /// <param name="statusCode">int.</param>
+        /// <param name="message">The Message.</param>
+        /// <param name="statusCode">The Message StatusCode.</param>
         /// <returns>JsonResult.</returns>
-        public static JsonResult CreateResult(string message, HttpStatusCode statusCode)
+        public static async Task<JsonResult> CreateResult(string message, HttpStatusCode statusCode)
         {
-            JsonResult res = new (new ErrorResult { Error = statusCode, Message = message })
-            {
-                StatusCode = (int)statusCode,
-            };
+           return await Task.Run(() =>
+           {
+               JsonResult res = new (new ErrorResult { Error = statusCode, Message = message })
+               {
+                   StatusCode = (int)statusCode,
+               };
 
-            return res;
+               return res;
```

Comment:
This method looks like it only returns an error, but could be passed good data and a 200.  If that's it's purpose then it should be renamed.

Reply:
Good point!, method was renamed along callers updated.